### PR TITLE
Added review changes and some validations

### PR DESCRIPTION
**Pull request by dnwick**

### CODE REVIEW

File: components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.ui/org.wso2.carbon.event.output.adapter.ui.endpoint/src/main/java/SuperTenantSubscriptionEndpoint.java [hunk 1]
```diff
@@ -54,7 +53,7 @@ public void onOpen (Session session, @PathParam("streamname") String streamName
             log.debug("WebSocket opened, for Session id: "+session.getId()+", for the Stream:"+streamName);
         }
         PrivilegedCarbonContext carbonContext = PrivilegedCarbonContext.getThreadLocalCarbonContext();
-        carbonContext.setTenantId(-1234);
+        carbonContext.setTenantId(carbonContext.getTenantId());
         tenantId = carbonContext.getTenantId();
```
Comment (mohanvive) [inline components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.ui/org.wso2.carbon.event.output.adapter.ui.endpoint/src/main/java/SuperTenantSubscriptionEndpoint.java]: Damith, I have an issue in above change.. If tenant id is there in the carbon context then why you are getting that and again setting that to carbon context.. What is expected here ?  I think there is an issue here.

File: components/event-publisher/org.wso2.carbon.event.publisher.core/src/main/java/org/wso2/carbon/event/publisher/core/internal/type/wso2event/WSO2EventOutputMapper.java [hunk 2]
```diff
@@ -74,6 +72,40 @@ public WSO2EventOutputMapper(EventPublisherConfiguration eventPublisherConfigura
...
+        this.inputStreamDefinition = inputStreamDefinition;
+
+        //try {
+        wso2EventOutputMapping = (WSO2EventOutputMapping) eventPublisherConfiguration.getOutputMapping();
+        if (!wso2EventOutputMapping.isCustomMappingEnabled()) {
+            //outputStreamDefinition = new StreamDefinition(outputStreamName, outputStreamVersion);
+            //outputStreamDefinition.setMetaData(inputStreamDefinition.getMetaData());
+            //outputStreamDefinition.setCorrelationData(inputStreamDefinition.getCorrelationData());
+            //outputStreamDefinition.setPayloadData(inputStreamDefinition.getPayloadData());
+
```
Comment (mohanvive) [inline components/event-publisher/org.wso2.carbon.event.publisher.core/src/main/java/org/wso2/carbon/event/publisher/core/internal/type/wso2event/WSO2EventOutputMapper.java]: Why above code segments commented ?

File: components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.ui/org.wso2.carbon.event.output.adapter.ui.endpoint/src/main/java/SuperTenantSubscriptionEndpoint.java (hunk 1 above)
Comment (dnwick) [inline components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.ui/org.wso2.carbon.event.output.adapter.ui.endpoint/src/main/java/SuperTenantSubscriptionEndpoint.java]: tenant Id I have to change. I understand you can getting it from tenant flow and  No need to pass it. I kept it for now. I will make the change once I review it with you or suho on Monday

File: components/event-publisher/org.wso2.carbon.event.publisher.core/src/main/java/org/wso2/carbon/event/publisher/core/internal/type/wso2event/WSO2EventOutputMapper.java (hunk 2 above)
Comment (dnwick) [inline components/event-publisher/org.wso2.carbon.event.publisher.core/src/main/java/org/wso2/carbon/event/publisher/core/internal/type/wso2event/WSO2EventOutputMapper.java]: We removed the Stream definition passing in wso2 event mapping in core. And corresponding changes are done in wso2 event output publisher. I was told to keep it commented out and do the change.

**Pull request merged**